src/pages/meetingsPage.jsx

```diff
@@ -0,0 +1,113 @@
+import FusionLogo from '../assets/fusionLogo.png';
+import PlaceholderPic from '../assets/placeholderPic.png';
+
+export function MeetingsPage() {
+    return (
+        <div style={{
+            display: 'flex',
+            width: '1275px',
+            alignItems: 'flex-start',
+            alignContent: 'flex-start',
+            gap: '124px 231px',
+            flexWrap: 'wrap',
+            margin: 'auto',
+            padding: '2px',
+        }}>
+            <div className="flex flex-col w-full h-screen bg-white">
+                <div className="flex w-full justify-end h-1/6">
+                    <div className="flex flex-row h-14 gap-x-4 pr-7 pt-5">
+                    </div>
+                </div>
+                <div className="relative flex flex-col items-center justify-start h-5/6 pt-10 px-20">
+                    <h1 style={{ fontSize: '40px', letterSpacing: '8px'}} className="font-inter font-medium tracking-wider">GENERAL MEETINGS</h1>
+                    <p style={{ fontSize: '20px', margin: '40px'}} className="font-inter font-normal mt-10 text-center">
```

Comment:
Due to the possibilities of individuals being in dark mode, please specifiy the color of the text, pls do this for all the text within this page